CHANGELOG.md

```diff
@@ -201,7 +201,9 @@
 
 ### refactor
 
-* refactor text-transform functions ([#225](https://github.com/gravity-ui/page-constructor/issues/225))
+* refactor text-transform functions ([#225](https://github.com/gravity-ui/page-constructor/issues/225)):
+  * delete transformFootnotes, because remove component Footnotes ([#343](https://github.com/gravity-ui/page-constructor/issues/343))
+  * now transformBlocks not import, use `contentTransformer`
```

Comment:
Maybe something like that?
removed function `transformBlocks`, use `contentTransformer` instead

Reply:
done